### PR TITLE
Accessibility

### PR DESCRIPTION
**Pull request by mirouhml**
- Page titles - no issues found.
- Image text alternatives  - no issues found.
- Text headings - issues found and fixed (changed some h2 headings to h3 headings).
- Color contrast  - no issues found.
- Resize  - no issues found.
- Interaction  - no issues found.
- Moving content  - no issues found.
- Multimedia  - no issues found.
- The basic structure of the page  - no issues found.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -8,6 +8,14 @@
...
     <link href="https://fonts.googleapis.com/css2?family=Crete+Round&family=Inter&family=Poppins&family=Roboto&display=swap" rel="stylesheet">
+    <!-- Global site tag (gtag.js) - Google Analytics -->
+    <script async src="https://www.googletagmanager.com/gtag/js?id=G-HV2JL7ZN8B"></script>
+    <script>
+    window.dataLayer = window.dataLayer || [];
+    function gtag(){dataLayer.push(arguments);}
+    gtag('js', new Date());
+    gtag('config', 'G-HV2JL7ZN8B');
+    </script>
     <title>Portfolio</title>
```
Comment (andersonlebon) [inline index.html]: - [ ] Your report is looking good however after a deep search I have found something that you should also consider and include in your report.
### Page titles `<title>Portfolio</title>`.
It adequately and briefly describes the content of the page. For example:
#### Poor titles:
- Microverse project | Your portfolio.
- Portfolio project
#### Better page titles:
- Mark's portfolio
- Mark full-stack web developer

**Review by andersonlebon — CHANGES_REQUESTED** [review]
### Hi @mirouhml  and @Alex
Good job so far! I know you have worked hard on making this milestone But  There are some issues that you still need to work on to go to the next project but you are almost there!

almost-there
## To highlight :champagne: 
- All linters are passing :heavy_check_mark: 
- Your repport is well structured.
## Required Changes ♻️

- [ ] Kindly check comments under the review.

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html (hunk 1 above)
Comment (harlexkhal) [reply index.html]: You tagged the wrong Alex @andersonlebon, I'm the one you should have tagged. thanks for the review

**Review by julie-ify — APPROVED** [review]
## Hi team, 

# Highlights
- Good job implementing the required changes from the previous reviewer ✔️ 

# Status: Approved ✔️ 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 

# Congratulations! 🎉 

Approved emoji

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**